Models/CourseModel.php

```diff
@@ -0,0 +1,568 @@
+<?php
+declare(strict_types=1);
+
+namespace Models;
+
+/**
+ * @package Models
+ */
+class CourseModel extends DataModel
+{
+    const INACTIVE_COURSE = 0;
```

Comment:
please declare scope of visibility for the constant. Public, protected or private